app/components/preprint-status-banner/component.js

```diff
@@ -257,6 +261,9 @@ export default Component.extend({
             this.set('decision', this.get('submission.reviewsState'));
             this.set('reviewerComment', this.get('initialReviewerComment'));
         },
+        closeReviewerFeedback() {
+            this.set('userHasEnteredReview', this.get('userActivity'));
```

Comment:
Because of how this is determined, and because "Accept" is selected by default, there might be a minor bug: If you go to a pending submission, then open and close the dropdown without changing anything, `userHasEnteredReview` will be true.